mantle/kola/harness.go

```diff
@@ -544,12 +544,12 @@ func RunUpgradeTests(patterns []string, pltfrm, outputDir string, propagateTestE
 
 // externalTestMeta is parsed from kola.json in external tests
 type externalTestMeta struct {
-	Architectures   string   `json:",architectures,omitempty"`
```

Comment:
Why there is a comma in the beginning? I searched it may be used for transferring from string to int, but it's not the case here. Golang-ci lint shows `SA5008: unknown JSON option`. Is it ok if we drop the comma?

Reply:
Correct, there should be no leading comma (i.e. it should be `json:"architectures,omitempty"`)

Reply:
Got it. Thank you!